GUI/DeusEx/Classes/MenuScreenRandoOptionsGameplay.uc

```diff
@@ -25,6 +25,10 @@ function CreateChoices()
     CreateChoice(class'MenuChoice_ConfirmNoteDelete');
     CreateChoice(class'MenuChoice_FixGlitches');
     CreateChoice(class'MenuChoice_AutoLamps');
+    CreateChoice(class'MenuChoice_RefuseUseless');
```

Comment:
item refusal only works in vanilla, move these lines into the if statement above, same thing for `GUI/DeusEx/Classes/MenuScreenRandoOptions.uc`